content/articles/getting-started-ssl-certificates.markdown

```diff
@@ -18,42 +18,55 @@ An [SSL certificate](/articles/ssl-certificates) is required to enable SSL on yo
 
 Getting a new SSL certificate is a multi-step process and involves several parties: the customer (you), DNSimple, and the [certificate authority](/articles/what-is-certificate-authority). For this reason, [this is not a real-time process](/articles/how-long-to-issue-ssl-certificate).
 
-## SSL certificate requirements {#requirements}
+## Before you start {#requirements}
 
 Before starting, take a moment to review how you will use the certificate and be sure to have access to the server configuration, or a feature to install a custom certificate on your server.
 
 If you are not familiar with SSL certificates, you may want to learn more about the different [SSL certificate types](/articles/ssl-certificates-types).
 
-Here are some questions you should be able to answer:
+In some limited instances, you may need to use a [custom CSR](/articles/what-is-csr) to acquire a new certificate. Check your web server's documentation to determine if you do.
```

Comment:
For this section, I suggest mentioning that we create a CSR for you regardless. If someone is uncertain what a CSR is, they may see "submit CSR" in their docs and automatically assume they need a custom one.